Answers.md

```diff
@@ -2,10 +2,24 @@
 
 1.  What is React JS and what problems does it try and solve?
 
+React JS is a JavaScript library made for the sole purpose of building dynamic user interfaces. It attempts to solve
+the issue where if you have a ton of components, you can use these components in your UI repetitively or dynamically.
+

```

Comment:
It also saves resources by maintaining a virtual DOM and only rendering changes to the actual DOM instead of constantly rerendering the actual DOM